packages/sku/config/eslint/index.js

```diff
@@ -0,0 +1,27 @@
+// @ts-check
+// @ts-expect-error `eslint-config-seek` has no types yet
+const eslintConfigSeek = require('eslint-config-seek');
+const { importOrderConfig } = require('./importOrder.js');
+const { createEslintIgnoresConfig } = require('./ignores.js');
+const {
+  eslintDecorator,
+  eslintIgnore,
+  languages,
+  paths: { relativeTarget },
+} = require('../../context/index.js');
```

Comment:
**question:** Is the barrel file not picked up here?

Also I don't believe the `.js` filetypes are needed here. Correct me if I'm wrong.

Reply:
My IDE auto-completed the path, and I guess it chose to be explicit. The `.js` isn't needed, but if we start writing ESM then it will be necessary, though I guess that can also be avoided if we use a bundler.

The extension on the import actually result in faster module resolution because node doesn't have to figure the extension. But overall it's not a huge deal in this codebase so it's not something I pay much attention to.